packages/@aws-cdk/aws-redshift/lib/cluster-ref.ts

```diff
@@ -0,0 +1,52 @@
+import * as ec2 from '@aws-cdk/aws-ec2';
```

Comment:
I would either call this file `icluster.ts` instead, or just put all of this stuff into `cluster.ts`, which is what we usually end up doing in similar situations :).